src/components/Cart/CartItem.jsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/prop-types */
+import trash from "../../images/trash.png";
+
+const formatCurrency = new Intl.NumberFormat(undefined, {
+  style: "currency",
+  currency: "USD",
+});
+
+const MAX_QUANTITY = 31;
+const MIN_QUANTITY = 1;
+
+function CartItem({ id, quantity, title, price, updateItem, removeItem }) {
+  return (
+    <li key={id} className="cart__item">
+      <div className="cart__item-info">
+        <h4>{title}</h4>
+        <span>
+          Quantity:
+          <input
+            id="item__quantity"
+            className="count__quantity"
+            type="number"
+            value={quantity}
+            min={MIN_QUANTITY}
+            max={MAX_QUANTITY}
+            onChange={(e) => {
```

Comment:
This should be extracted into it's own callback

Reply:
Done!